File: swift/name.swift
// var name: String = "Swift";
// name = "42"; // Error: Cannot assign value of type 'Int' to type 'String'

// print(name);


import Foundation

print("Enter a number:")
if let input = readLine(), let number = Int(input) {
    if number > 35 {
        print("Too old")
    } else {
        print("OK")
    }
    // print("You entered: \(number)")
} else {
    print("Invalid input")
}

File: python/name.py
import sys

# name ="David";
# name = 42;

# print(name);

if len(sys.argv) > 1:
    try:
        number = int(sys.argv[1])
        if number > 35:
            print("TOO OLD")
        else:
            print("TOO YOUNG")
    except ValueError:
        print("Please enter a valid number.")
else:
    print("No number entered.")